File: rest/member.py
import json

class Member():

    _identifier = 0
    _firstname = ""
    _lastname = ""
    _handicap = 0

    def __init__(self, identifier, firstname, lastname, handicap):
        self._identifier = identifier
        self._firstname = firstname
        self._lastname = lastname


    def jsonify(self):
        data = { 'identifier' : self._identifier,
                 'firstname' : self._firstname,
                 'lastname' : self._lastname,
                 'handicap' : self._handicap }
        return json.dumps(data)


    @property
    def identifier(self):
        return self._identifier

    @property
    def firstname(self):
        return self._firstname

    @property
    def lastname(self):
        return self._lastname

    @property
    def score(self):
        return self._score

File: app/static/member.js
function Member(member_id, member_f_name, member_l_name, member_handicap) {
  this.member_id = member_id;
  this.member_f_name = member_f_name;
  this.member_l_name = member_l_name;
  this.member_handicap = member_handicap;

  this.get_member_id = function() {
    return this.member_id;
  }
  this.get_member_f_name = function() {
    return this.member_f_name;
  }
  this.get_member_l_name = function() {
    return this.member_l_name;
  }
  this.get_member_handicap = function() {
    return this.member_handicap;
  }
  this.set_member_id = function(member_id) {
    this.member_id = member_id;
  }
  this.set_member_f_name = function(member_f_name) {
    this.member_f_name = member_f_name;
  }
  this.set_member_l_name = function(member_l_name) {
    this.member_l_name = member_l_name;
  }
  this.set_member_handicap = function(member_handicap) {
    this.member_handicap = member_handicap;
  }

  this.get_member_fullname = function() {
    return this.member_f_name + ' ' + this.member_l_name;
  }
  this.jsonify = function() {
    var jsonData = {
      "firstname" :this.get_member_f_name(),
      "lastname"  :this.get_member_l_name(),
      "handicap"  :this.get_member_handicap(),
      "identifier":this.get_member_id()
    }
    return JSON.stringify(jsonData);
  }
}
